### PR TITLE
Reimplement filterbar on grid view

### PR DESCRIPTION
**Pull request by frederike-ramin**
Because of huge merge conflicts: Replacement for #84.

### CODE REVIEW

File: src/GridView/CardGrid.js [hunk 1]
```diff
@@ -13,6 +13,11 @@ import config from './../config/config';
 import Utils from './../Utils/Utils';
 
 class CardGrid extends ConnectionComponent {
+	constructor(props) {
```
Comment (hpibptlab) [inline src/GridView/CardGrid.js]: MAJOR Remove the unused function parameter "props". [rule](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)

**Comment by hpibptlab** [comment]
SonarQube analysis reported 1 issue
* ![MAJOR][MAJOR] 1 major

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'

File: src/GridView/GridView.js [hunk 2]
```diff
@@ -12,8 +12,12 @@ import Header from "../Header";
 class GridView extends ConnectionComponent {
 	constructor() {
```
Comment (MaximilianV) [inline src/GridView/GridView.js:13]: Please use tabs for indentation (multiple lines in constructor affected)

**Review by MaximilianV — CHANGES_REQUESTED** [review]
Looks good, please check indentation as mentioned in comment

**Review by MaximilianV — APPROVED** [review]

**Pull request merged**